asset/modules/mailLib/mailreply.py

```diff
@@ -0,0 +1,37 @@
+from email.mime.multipart import MIMEMultipart
+from email.mime.text import MIMEText
+import smtplib
+ 
+
+def shootMail(details): 
+    # create message object instance
+    msg = MIMEMultipart()
+    message = details['Message']
+    
+    # setup the parameters of the message
+    password = details['Password']
+    msg['From'] = details['Sender']
+    msg['To'] = details['Receiver']
+    msg['Subject'] = details['Subject']
+ 
+ 
+    #create server
+    server = smtplib.SMTP('smtp.gmail.com: 587')
```

Comment:
I don't have that much knowledge of the gmail server. I just want to ask if this server will be the same for everyone? :thinking:

Reply:
According to my knowledge, the server will be the same for everyone.

Reply:
It will be.